lib/pages/login/login_page.dart

```diff
@@ -45,7 +47,18 @@ class LoginPage extends StatelessWidget {
                 style: Theme.of(context).textTheme.bodyLarge,
               ),
             ),
-            const GoogleSignInButton()
+            Column(
+              children: [
+                const GoogleSignInButton(),
+                // Toggle button to signup page
+                TextButton(
+                  onPressed: () {
+                    Navigator.pushNamed(context, SignUpPage.routeName);
+                  },
+                  child: const Text("Don't have an account? Sign Up"),
```

Comment:
Underline Sign Up
If you want you can use RichText

Reply:
Fixed [here](https://github.com/gdsc-nits-org/efficacy_admin/pull/20/commits/e454175105909e82cbb853b1e0f02321f332daad)